tests/test_main.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from subprocess import CompletedProcess  # noqa: S404
+from typing import TYPE_CHECKING
+
+from pipdeptree.__main__ import main
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    import pytest
+    from pytest_mock import MockFixture
+
+
+def test_main_log_resolved(tmp_path: Path, mocker: MockFixture, capsys: pytest.CaptureFixture[str]) -> None:
+    cmd = ["", "--python", "auto"]
+    mocker.patch("pipdeptree.__main__.sys.argv", cmd)
+
+    mocker.patch("pipdeptree._detect_env.detect_active_interpreter", return_value=str(tmp_path))
+    mocker.patch("pipdeptree._detect_env.os.environ", {"VIRTUAL_ENV": str(tmp_path)})
+    mocker.patch("pipdeptree._detect_env.Path.exists", return_value=True)
```

Comment:
Since we are now mocking `detect_active_interpreter()` it looks like we can throw out these two patch calls.